command.go

```diff
@@ -31,6 +31,12 @@ import (
 // FParseErrWhitelist configures Flag parse errors to be ignored
 type FParseErrWhitelist flag.ParseErrorsWhitelist
 
+// Structure to manage groups for commands
+type Group struct {
+	Group string
```

Comment:
How about calling this field `ID` or something like that?  I find that `Group.Group` is confusing.

Reply:
I'll change it to `Id`.